dotNet/initialize.js

```diff
@@ -70,14 +71,17 @@ module.exports = function (tracker, test) {
 	addSyntax('return', ({ value }) => `return ${value}`)
 
 	addHook('deps', `using NUnit.Framework;`)
+	addHook('deps', `using Applitools.Tests.Utils;`)
+	addHook('deps', `using Applitools.Generated.Utils;`)
+	addHook('deps', `using Applitools.Utils.Geometry;`)
 	if (mobile) {
 		addHook('deps', `using Applitools.Appium.GenericUtils;`)
+		addHook('deps', `using OpenQA.Selenium;`)
```

Comment:
this line is the same as line 83. I think it'll be safe to take outside the `if`

Reply:
Done